webclient/app/src/app/features/apps/entities/entityDiagram/EntityDiagram.jsx

```diff
@@ -105,21 +150,54 @@ function EntityDiagram(props) {
             }
 
             return (
-                    <Draggable
-                            axis={"both"}
-                            onStop={(update, draggableData) => {
-                                savePosition(entity, index, draggableData);
-                            }}
-                            key={entity.id + index + ""}
-                            defaultClassName={entityDiagramStyles.draggable}
-                            position={entityCardPosition}
-                            disabled={!editable}
-                    >
-                        <div>
-                            <EntityCard entity={entity} onEdit={setSelectedEntityId} handleDelete={deleteEntity}
-                                        editable={editable}/>
-                        </div>
-                    </Draggable>
+                <Draggable
+                    axis={"both"}
+                    onStop={(update, draggableData) => {
+                        savePosition(entity, index, draggableData);
+                    }}
+                    key={entity.id + index + ""}
+                    defaultClassName={entityDiagramStyles.draggable}
+                    position={entityCardPosition}
+                    disabled={!editable}
+                >
+                    <div>
+                        <EntityCard entity={entity} onEdit={setSelectedEntityId} handleDelete={deleteEntity}
+                            editable={editable}/>
+                    </div>
+                </Draggable>
+            );
+        });
+    }
+
+    function renderEnums() {
+        if (entities.length === 0 && enums.length === 0 ) {
+            return;
+        }
+        return enums.map((enumDef, index) => {
+            const enumDefPostiton = {x: 0, y: 0};
+
+            if (enumDef.position) {
+                const {positionX, positionY} = enumDef.position;
+                enumDefPostiton.x = positionX;
+                enumDefPostiton.y = positionY;
+            }
+
+            return (
+                <Draggable
```

Comment:
Ich weiß nicht, ob das so gut ist, dass die Enums noch als Karten in dem Diagram auftauchen. Würde das seperat ablegen tatsächlich. Ich glaube das könnte sonst zu viel auf dem Bildschirm werden.

Reply:
Könnten wir das nochmal diskutieren? Evtl könntest du nochmal ein Screenshot an den PR anhängen. Habe gerade ne schlechte vorstellung, wie das gedacht ist

Reply:
Ich habe ein Screenshot angefügt. In JHipster ist das ähnlich dargestellt. Ich finde die gehören dazu und ich würde auch ungern viel Aufwand deswegen betreiben.

Reply:
Okey, würde aber perspektivisch gerne noch das Design der Karte anpassen wollen. Ich persönlich finde, dass die nicht zu den anderen Komponenten von dem Layout etc passt. Wäre das ok?

Reply:
Hätte ein Ticket angelegt - schreibe gerne, ob das für dich io ist oder nicht:
https://dev.azure.com/starwit/lj-projectbuilder/_backlogs/backlog/lj-projectbuilder%20Team/Features/?workitem=289